apps/base-docs/docs/pages/identity/smart-wallet/guides/paymasters.mdx

```diff
@@ -1,52 +1,75 @@
+import { Callout } from "vocs/components"
+
 # Paymasters (Sponsored Transactions)
 
 One of the biggest UX enhancements unlocked by Smart Wallet is the ability for app developers to sponsor their users' transactions. If your app supports Smart Wallet, you can start sponsoring your users' transactions by using [standardized paymaster service communication](https://erc7677.xyz) enabled by [new wallet RPC methods](https://eip5792.xyz).
 
 The code below is also in our [Wagmi Smart Wallet template](https://github.com/wilsoncusack/wagmi-scw/).
 
-## Using Wagmi + Permissionless in a Next.js app
+<Callout type="warning">
+**About The Hooks Used Below**
 
-:::warning
 The `useWriteContracts` and `useCapabilities` hooks used below rely on new wallet RPC and are not yet supported in most wallets.
 It is recommended to have a fallback function if your app supports wallets other than Smart Wallet.
-:::
+</Callout>
+
+## Using Wagmi/Viem in a Next.js app
 
 ::::steps
 
 ### Choose a paymaster service provider
 
-As a prerequisite, you'll need to obtain a paymaster service URL from a paymaster service provider. To be compatible with Smart Wallet, the paymaster provider you choose must be ERC-7677-compliant.
+As a prerequisite, you'll need to obtain a paymaster service URL from a paymaster service provider.
+
+We recommend the [Coinbase Developer Platform](https://www.coinbase.com/developer-platform) paymaster,
+currently offering up to $15k in gas credits as part of the [Base Gasless Campaign](/identity/smart-wallet/introduction/base-gasless-campaign).
 
-We recommend the [Coinbase Developer Platform](https://www.coinbase.com/developer-platform) paymaster, currently offering up to $15k in gas credits as part of the Base Gasless Campaign(/identity/smart-wallet/introduction/base-gasless-campaign).
-You can find a full list of ERC-7677-compliant paymaster services [here](https://www.erc7677.xyz/ecosystem/paymasters).
+Once you have signed up for Coinbase Developer Platform, you get your Paymaster service URL by navigating to Onchain Tools > Paymaster as shown below:
+
+<div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+  <img
+    alt="Paymaster CDP"
+    src="/images/smart-wallet/PaymasterCDP.png"
+    width="1000"
+    loading="lazy"
+  />
+  <em>How to get your Paymaster service URL</em>
+</div>
```

Comment:
I think we should change the wording here to be like "We'll use Coinbase Developer Platform as a paymaster service provider" - having the premise that you can bring any provider muddies the doc